propagation_test.go

```diff
@@ -30,13 +30,20 @@ import (
 	"github.com/opentracing/opentracing-go/ext"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/uber/jaeger-lib/metrics"
 )
 
+func initMetrics() (*metrics.LocalBackend, *Metrics) {
+	b := metrics.NewLocalBackend(0)
+	factory := metrics.NewLocalFactory(b)
+	return b, NewMetrics(factory, nil)
+}
+
 func TestSpanPropagator(t *testing.T) {
 	const op = "test"
 	reporter := NewInMemoryReporter()
-	stats := NewInMemoryStatsCollector()
-	metrics := NewMetrics(stats, nil)
+	stats, metrics := initMetrics()
```

Comment:
what if we call stop() from initMetrics()?  We're not interested in periodic reporting during tests. Even better if this was an option when creating local backend